pkg/controller/logstorage/esgateway.go

```diff
@@ -129,7 +131,7 @@ func (r *ReconcileLogStorage) createEsGateway(
 		TrustedBundle: trustedBundle,
 	})
 
-	for _, comp := range []render.Component{esGatewayComponent, certificateComponent} {
+	for _, comp := range []render.Component{certificateComponent, esGatewayComponent} {
```

Comment:
Changed the order so the secrets and cm are rendered before the deployments etc. The namespace is already rendered by logstorage.go